### PR TITLE
Feature/sc 9159 news typescript frontend

### PR DESCRIPTION
**Pull request by mwolowyk-cap**
# Short Description

Activate old news UIs. 
Use new v3 backend routes 

## Links to Ticket and related Pull-Requests
https://ticketsystem.hpi-schul-cloud.org/browse/SC-9159

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/organisms/FormNews.vue [hunk 1]
```diff
@@ -119,16 +120,24 @@ export default Vue.extend({
...
+		} {
 			const title = this.data.title
 				? undefined
 				: this.$ts("components.organisms.FormNews.errors.missing_title");
 			const content = this.data.content
 				? undefined
 				: this.$ts("components.organisms.FormNews.errors.missing_content");
+			const date = this.data.date.date
+				? undefined
+				: this.$ts("components.organisms.FormNews.errors.missing_date");
```
Comment (mwolowyk-cap) [inline src/components/organisms/FormNews.vue:136]: Don't change the domain logic. Put default date = now

**Comment by mwolowyk-cap** [comment]
@schul-cloud-bot update screenshots

**Comment by mwolowyk-cap** [comment]
@schul-cloud-bot update screenshots

**Review by pkleybolte — CHANGES_REQUESTED** [review]
Unit Test are missing for NewsEditView, NewsDetailView, all of edit.vue and index.vue

**Comment by mwolowyk-cap** [comment]
> Unit Test are missing for NewsEditView, NewsDetailView, all of edit.vue and index.vue

@pkleybolte These components are not in use yet. They are currently disabled. Before usage they have to be tested.

**Pull request closed without merging**